frontend/lib/src/util/fuzzySearch.test.ts

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2025)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fzy from "./fuzzySearch"
+
+const score = fzy.score
+const positions = fzy.positions
+
+const SCORE_MIN = fzy.SCORE_MIN
+const SCORE_MAX = fzy.SCORE_MAX
+
+const SCORE_GAP_LEADING = fzy.SCORE_GAP_LEADING
+const SCORE_GAP_TRAILING = fzy.SCORE_GAP_TRAILING
+const SCORE_GAP_INNER = fzy.SCORE_GAP_INNER
+const SCORE_MATCH_CONSECUTIVE = fzy.SCORE_MATCH_CONSECUTIVE
+const SCORE_MATCH_SLASH = fzy.SCORE_MATCH_SLASH
+const SCORE_MATCH_CAPITAL = fzy.SCORE_MATCH_CAPITAL
+const SCORE_MATCH_DOT = fzy.SCORE_MATCH_DOT
+
+/* score(needle, haystack) */
+describe("score", () => {
+  it("should prefer starts of words", function () {
+    /* App/Models/Order is better than App/MOdels/zRder  */
```

Comment:
```suggestion
    /* app/models/order is better than app/models/zrder  */
```

nit: Could you leave a comment here about the _why_ it is better? I assume its because the `o` in `order` comes before the `z` in `zrder`.